src/Chapter1.hs

```diff
@@ -615,7 +627,10 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = let d0 = mod n 10
+                 d1 = (mod (div n 10) 10)
+             in d0 + d1
```

Comment:
On a side note, there is a standard function as well that combines both `div` and `mod` at once: `divMod`.

It returns the pair, so you can pattern match on that and get both variabless at once:

```haskell
let (a, b) = abs n `divMod` 10
```



That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂